pcieutil/main.py

```diff
@@ -41,22 +41,17 @@ def print_result(name, result):
 
 def load_platform_pcieutil():
     global platform_pcieutil
-    global platform_plugins_path
+    global platform_path
 
     # Load platform module from source
     try:
         platform_path, _ = device_info.get_paths_to_platform_and_hwsku_dirs()
-        platform_plugins_path = os.path.join(platform_path, "plugins")
-        sys.path.append(os.path.abspath(platform_plugins_path))
-        from pcieutil import PcieUtil
+        sys.path.append(os.path.abspath(platform_path))
```

Comment:
Why do we need to append `platform_path` to `sys.path`? We're not loading Python modules from here, correct? Only the pcie.yaml file resides there, right?

Reply:
correct.